src/OpenTelemetry.Extensions.Docker/CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 ## Unreleased
 
+## 1.1.0-beta.1
```

Comment:
This package has never had a stable release. Do you want to just name this version `1.0.0-beta.2` instead?

Reply:
I'm not against it. I'm not sure, if we should follow SemVer - but it suggest to change "MINOR version when you add functionality in a backwards compatible manner". CGroup V2 is new functionality, so I increased minor version.

Please confirm if you still think it would be better to have `1.0.0-beta.2` and I'll change it.

Reply:
Please use 1.0.0-beta.2.
After a stable package is released, we can (and should) bump minor version for additional functionality.